lib/components/narrative/tabbed-itineraries.js

```diff
@@ -2,14 +2,24 @@ import coreUtils from '@opentripplanner/core-utils'
 import PropTypes from 'prop-types'
 import React, { Component } from 'react'
 import { Button } from 'react-bootstrap'
+import { FormattedMessage, FormattedNumber } from 'react-intl'
 import { connect } from 'react-redux'
+import styled from 'styled-components'
 
 import * as narrativeActions from '../../actions/narrative'
 import { ComponentContext } from '../../util/contexts'
 import { getActiveSearch, getRealtimeEffects } from '../../util/state'
+import { FormattedDuration } from '../util/format-duration'
+import { FormattedTime } from '../util/format-time'
 
-const { calculateFares, calculatePhysicalActivity, getTimeZoneOffset } = coreUtils.itinerary
-const { formatDuration, formatTime, getTimeFormat } = coreUtils.time
+const { calculateFares, calculatePhysicalActivity } = coreUtils.itinerary
+
+const Bullet = styled.span`
```

Comment:
It is a trick to avoid defining a `const Bullet = () => <span>•</span>` that would still need to be styled for margins. The CSS `content` attribute only works with pseudo-elements.